cmd/gateway-main.go

```diff
@@ -365,8 +365,10 @@ func gatewayMain(ctx *cli.Context, backendType gatewayBackend) {
 		// routes them accordingly. Client receives a HTTP error for
 		// invalid/unsupported signatures.
 		setAuthHandler,
-		// Add new handlers here.
+		// Add HTTP logging handler,
+		setLoggingHandler,
```

Comment:
having it as part of the middleware will be useful in enabling logging using `mc admin` command without having to restart minio.